resources/core/charts/cluster-users/templates/rbac-roles.yaml

```diff
@@ -241,6 +249,28 @@ aggregationRule:
       rbac.authorization.kyma-project.io/aggregate-to-kyma-backup-admin: "true"
 rules: []
 
+---
+################################################################################
+# Namespace Admin role
+# kyma-namespace-admin = kyma-essentials + kyma-namespace-create
+################################################################################
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kyma-namespace-admin
```

Comment:
true namespace-admin is this role + roleBinding to kyma-admin

Reply:
changed